pkg/typedef/columns_test.go

```diff
@@ -131,6 +133,42 @@ func TestMarshalUnmarshal(t *testing.T) {
 	}
 }
 
+func TestMarshalUnmarshalSchemaExample(t *testing.T) {
```

Comment:
It should be test for `readSchema`, please use it in the test and move test into package of `readSchema`

Reply:
Done